src/value.cc

```diff
@@ -680,6 +697,7 @@ bool V8ToGIArgument(GITypeInfo *type_info, GIArgument *arg, Local<Value> value,
     case GI_TYPE_TAG_VOID:
         if (g_type_info_is_pointer(type_info)) {
             arg->v_pointer = PointerFromWrapper(value);
+            // XXX what to do with GI_TRANSFER_EVERYTHING
```

Comment:
Because the value is nominally owned by the V8 object, `GI_TRANSFER_EVERYTHING` means we need to copy or increase reference of the object. For `GI_TYPE_TAG_VOID` I just don't have a way to copy it. At least, not that I can see in this project's source code.

Reply:
Is that case ever encountered? Put an `ERROR("Case should not be reached. Please report this to https://github.com/romgrk/node-gtk/issues")` guard, if it is we'll get a report.

Reply:
The `ERROR()` is put in this location, if transfer is `GI_TRANSFER_EVERYTHING`.